src/main/java/com/velox/sloan/cmo/workflows/kapalibrary/IndexBarcodeToSampleAutoAssigner.java

```diff
@@ -40,52 +41,64 @@ public IndexBarcodeToSampleAutoAssigner() {
 
     @Override
     protected boolean shouldRun() throws Throwable {
-        return activeTask.getTask().getTaskOptions().containsKey("AUTOASSIGN INDEX BARCODES") && !activeTask.getTask().getTaskOptions().containsKey("_INDEXES_AUTO_ASIGNED");
+        return (activeTask.getTask().getTaskOptions().containsKey("AUTOASSIGN INDEX BARCODES") && !activeTask.getTask().getTaskOptions().containsKey("_INDEXES_AUTO_ASIGNED"))
+                || (activeTask.getTask().getTaskOptions().containsKey("TCRseq-IGO"));
```

Comment:
I don't think we need this to determine if the plugin should run

Reply:
Are we calling the same task option for TCRseq workflow?

Reply:
Correct, the task option value is different on TCRseq workflow, fixed.